remoulade/middleware/retries.py

```diff
@@ -67,6 +71,12 @@ def __init__(
         self.backoff_strategy = backoff_strategy
         self.jitter = jitter
         self.increase_priority_on_retry = increase_priority_on_retry
```

Comment:
add validation that we can use this if we have a quorum queue ? 
maybe add a has_priority or priority_enbaled property to Broker

Reply:
I think we should forbid to have both increase_priority_on_retry and escalation_queue_mapping